File: ListViewDemo/app/src/main/java/com/example/listviewdemo/MainActivity.java
package com.example.listviewdemo;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.AdapterView;
import android.widget.ArrayAdapter;
import android.widget.ListView;

import java.sql.Array;
import java.util.ArrayList;

public class MainActivity extends AppCompatActivity {

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        ListView myListView = findViewById(R.id.myListView);
        final ArrayList<String> allNames = new ArrayList<>();
        allNames.add("Harish");
        allNames.add("Manish");
        allNames.add("Ruchit");
        allNames.add("Mayank");
        allNames.add("Karan");

        ArrayAdapter<String> arrayAdapter = new ArrayAdapter<>(this,android.R.layout.simple_list_item_1,allNames);
        myListView.setAdapter(arrayAdapter);

        myListView.setOnItemClickListener(new AdapterView.OnItemClickListener() {
            @Override
            public void onItemClick(AdapterView<?> parent, View view, int position, long id) {
                    Log.i("Name",allNames.get(position));
            }
        });
;    }


}

File: images/app/src/main/java/com/example/images/MainActivity.kt
package com.example.images

import android.os.Build
import android.os.Bundle
import android.util.Log
import android.view.View
import android.widget.ImageView
import androidx.annotation.RequiresApi
import androidx.appcompat.app.AppCompatActivity


class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)
    }


    fun nextImgChange(view: View){
        var imageView = findViewById<ImageView>(R.id.imageView);
        imageView.setImageResource(R.drawable.husky1);
    }
}
